src/driver/mysql/MysqlDriver.ts

```diff
@@ -231,7 +231,7 @@ export class MysqlDriver implements Driver {
      */
     prepareHydratedValue(value: any, columnMetadata: ColumnMetadata): any {
         if (columnMetadata.type === Boolean) {
-            return value ? true : false;
```

Comment:
now another test is failing. However, i cannot find this test in the github-issues folder